### PR TITLE
Document `/source/{project_name}/{package_name}?cmd=updatepatchinfo` with OpenAPI

### PR DESCRIPTION
**Pull request by krauselukas**

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_updatepatchinfo.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    infos available through the issue trackers.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '403':
```
Comment (krauselukas) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_updatepatchinfo.yaml:14]: couldn't find a `400` case in my tests

**Comment by krauselukas** [comment]
[Sources - Package Command II (L)](https://trello.com/c/5pxIpBNy/2032-sources-package-command-ii-l)

File: src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_updatepatchinfo.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      description: Not Found
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Not Found:
+              value:
+                code: unknown_project
+                summary: 'Project not found: openSUSE:Maintenance:123'
```
[the remark this answers is not in the document]
Comment (krauselukas) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_updatepatchinfo.yaml]: @eduardoj yes, updated

[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_updatepatchinfo.yaml]: @krauselukas awesome!

**Review by eduardoj — APPROVED** [review]

**Pull request merged**